lms/views/content_item_selection.py

```diff
@@ -49,6 +49,10 @@
     schema=ContentItemSelectionLTILaunchSchema,
 )
 def content_item_selection(context, request):
+    request.find_service(name="application_instance").get().update_lms_data(
+        request.params
+    )
```

Comment:
This does make me realise that it might be worth moving `update_lms_data()` into the service because then `request.params` can be injected by the service factory rather than each view having to pass it in. For example it could be:

```python
request.find_service(name="application_instance").update()
```

Reply:
`request.find_service(name="application_instance").update()` That would make me scratch my head, "update how?, there's no params!"

I find explicitly passing the info in easier to understand but if you think this more pyramid-y happy to change it